ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/modal/v1/modal/_cq_template.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<jcr:root xmlns:jcr="http://www.jcp.org/jcr/1.0" xmlns:nt="http://www.jcp.org/jcr/nt/1.0" xmlns:cq="http://www.day.com/jcr/cq/1.0" xmlns:sling="http://sling.apache.org/jcr/sling/1.0" xmlns:fd="http://www.adobe.com/aemfd/fd/1.0"
+          jcr:primaryType="nt:unstructured"
+          jcr:title="Modal"
+          fieldType="panel">
+    <panelcontainer
+            fieldType="panel"
+            jcr:primaryType="nt:unstructured"
+            jcr:title="Panel"
+            sling:resourceType="core/fd/components/form/panelcontainer/v1/panelcontainer"
+            name="panelcontainer">
+        <fd:rules
+                jcr:primaryType="nt:unstructured"
+                validationStatus="valid"
+                visible="false()"/>
+        <fd:events
+                jcr:primaryType="nt:unstructured"/>
+    </panelcontainer>
+    <button
+            fieldType="button"
+            jcr:primaryType="nt:unstructured"
+            jcr:title="Open Modal"
+            sling:resourceType="core/fd/components/form/button/v1/button">
+        <fd:rules
+                jcr:primaryType="nt:unstructured"
+                validationStatus="valid"/>
+
+        <fd:events
+                jcr:primaryType="nt:unstructured"
+                click="dispatchEvent(panelcontainer, 'custom:setProperty', {visible : true()})"/>
```

Comment:
Have we tested it with 2 or more modals in the same form? Do we have such a use-case?

Reply:
multiple modals can be in the same form, but a modal can't be added inside a modal